packages/ListBoxWithTags/src/ListBoxWithTags.js

```diff
@@ -76,60 +36,65 @@ const renderTrigger = ({
+    return isOpen ? <sc.UpIcon isDisabled={isDisabled} /> : <sc.DownIcon isDisabled={isDisabled} />;
   }
 
+  const triggerProps = propsForTrigger();
   const a11yTextOptions = selectedOptions.map(item => {
     return tagLabelKey === null ? item.label : item[tagLabelKey];
   });
-
   const a11yText = selectedOptions.length === 0 ? t("listBoxWithTags.placeholder") : a11yTextOptions.join(", ");
 
   return (
-    <sc.Trigger
-      a11yText={t("listBoxWithTags.a11y_text_trigger", { options: a11yText })}
-      aria-controls={listBoxId}
-      aria-expanded={isOpen}
-      aria-haspopup
-      ref={refTrigger}
-      {...propsForTrigger()}
-      onClick={handleClick}
-      onKeyUp={handleKeyUp}
-      onKeyDown={handleKeyDown}
-      size={size}
-      allOptionsAreSelected={allOptionsAreSelected}
-    >
-      <Tags>
-        {selectedOptions.map(item => {
-          if (typeof renderTag === "function") {
-            return renderTag({ option: item, Tag, onRemove: handleRemove(item) });
-          }
-
-          const label = tagLabelKey === null ? item.label : item[tagLabelKey];
-
-          if (typeof label !== "string") {
-            throw Error(
-              `Your item ${JSON.stringify(
-                item
-              )} must include the attribute "label", or you must indicate which attribute should be rendered as the label via the "tagLabelKey" prop.`
+    <>
+      <sc.Trigger
+        {...triggerProps}
+        aria-controls={idListBoxContent}
+        allOptionsAreSelected={allOptionsAreSelected}
+        hasError={hasError}
+        isDisabled={isDisabled || isReadOnly}
+        isReadOnly={isReadOnly}
+        onClick={handleClick}
+        onKeyUp={handleKeyUp}
+        onKeyDown={handleKeyDown}
+        ref={refTrigger}
+        size={size}
+      >
+        <Tags>
+          {selectedOptions.map(item => {
+            if (typeof renderTag === "function") {
+              return renderTag({ option: item, Tag, onRemove: handleRemove(item) });
+            }
+
+            const label = tagLabelKey === null ? item.label : item[tagLabelKey];
+
+            if (typeof label !== "string") {
+              throw Error(
+                `Your item ${JSON.stringify(
+                  item
+                )} must include the attribute "label", or you must indicate which attribute should be rendered as the label via the "tagLabelKey" prop.`
+              );
+            }
+
+            return (
+              <Tag as="li" key={label} onRemove={handleRemove(item)} size={size}>
+                {label}
+              </Tag>
             );
-          }
-
-          return (
-            <Tag as="li" key={label} onRemove={handleRemove(item)} size={size}>
-              {label}
-            </Tag>
-          );
-        })}
-        {selectedOptions.length ? null : (
-          <sc.PlaceHolder>
-            <sc.PlaceHolderText>{t("listBoxWithTags.placeholder")}</sc.PlaceHolderText>
-          </sc.PlaceHolder>
```

Comment:
I don't know why this needs to have two elements – I've combined into a single `<sc.PlaceHolder>` and it seems to work fine.